generated/xamarin-partition-based-sync/realm-todo-dotnet/ItemPage.xaml.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System;
+using System.Linq;
+using RealmTemplateApp.Models;
+using Realms;
+using Xamarin.Forms;
+using Realms.Sync;
+using System.Collections.Generic;
+using Realms.Sync.Exceptions;
+
+namespace RealmTemplateApp
+{
+    public partial class ItemPage : ContentPage
+    {
+        private Realm itemsRealm;
+        private User user;
+        private IEnumerable<Item> _items;
```

Comment:
There seems to be an inconsistency with member variable naming - some getting `_` and others not. Should we pick one and stick with it?

Reply:
Yes, good point. All 3 are private fields; renamed to include the _.